models/hospital.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Hospital extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define associations here
+      // TODO
+    }
+  }
+  Hospital.init(
+    {
+      hospitalId: {
```

Comment:
One more naming convention suggestion: if we make the PRIMARY KEY attribute name ```id``` for each model, that will make the code more concise/less verbose...

I.e. instead of variable references like:
```
hospital.hospitalId
hospitalAdmin.hospitalAdministratorId
...
```
we would have:
```
hospital.id
hospitalAdmin.id
...
```

Reply:
@francisli I'm totally open on that. You all know the code consequences better than me. The foreign keys would, of course, still have the prefix, though - right?

Reply:
@francisli good idea, I'll update the models.

@billhoke keeping the prefix for foreign keys sounds good to me.

Reply:
@billhoke yes, foreign keys would still have the prefix... no changes in your export needs to be made, we can make the change just in the model source for convenience and still map to your fully prefixed primary key columns.